js/iframe-coordinator/elements/frame-router.ts

```diff
@@ -25,6 +42,15 @@ class FrameRouterElement extends HTMLElement {
     window.addEventListener("message", event => {
       this.router.ports.fromClient.send(event.data);
     });
+
+    // Subscribe to component out port to handle messages 
+    this.router.ports.toHost.subscribe(labeledMsg => {
+      if (labeledMsg.msgType === TOAST_REQUEST_MSG_TYPE) {
+        this.dispatchEvent(new CustomEvent(TOAST_REQUEST_EVENT_TYPE, {detail: labeledMsg.msg}));
```

Comment:
I am currently just dispatching this custom event.  Thoughts on if we should support an `onToastRequest` attribute and/or property?

Reply:
I also need to revisit the support for CustomEvent.  This constructor isn't supported in IE, but might be handled in a polyfill.